test/mocks/ringpop.go

```diff
@@ -8,6 +8,7 @@ import "github.com/uber/ringpop-go/events"
 import "github.com/uber/ringpop-go/forward"
 
 import "github.com/uber/ringpop-go/swim"
+
```

Comment:
turns out that we use the same version and that I am generating the white line as well now. Probably something to do with a change in the source file that this generates from. Nothing to do here.